lib/nodes.js

```diff
@@ -6,6 +6,10 @@ export const getNodes = async (fetchNodes = fetchNodesFromViewblock) => {
   let maxPage = Infinity
   for (let page = 0; page < maxPage; page++) {
     const res = await fetchNodes(page)
+    if (!res.ok) {
+      continue
+    }
```

Comment:
Let's print the status code and response body to console, to simplify troubleshooting.

```suggestion
    if (!res.ok) {
      const reason = await res.text().catch(err => err)
      console.log('fetchNodes(%s) failed - %s: %s', page, res.statusCode, reason)
      continue
    }
```